sktime/forecasting/sarimax.py

```diff
@@ -85,7 +85,9 @@ class SARIMAX(_StatsModelsAdapter):
         Whether or not to use exact diffuse initialization for non-stationary
         states. Default is False (in which case approximate diffuse
         initialization is used).
-    random_state : int, RandomState instance or None, optional ,
+    disp : bool, optional, default=False
+        Set to True to print convergence messages.
```

Comment:
It's the same message as in the `statsmodels` documentation and the same message other models in sktime, like `AutoETS`, have. Although I agree it's not completely clear, having a different description could make the user think (wrongly) that the parameter does something different than in the other models.

Reply:
> Although I agree it's not completely clear, having a different description could make the user think (wrongly) that the parameter does something different than in the other models.

I see - would you mind a separate PR then whre it's clarified? It is unclear what a "convergence message" is or whether it impacts other output prints. Just in case you know exactly what it does, I would have to guess or dig into code.

Side note, I think it shows that these docstrings were written before the advent of genAI...

Reply:
Sure, I'll make the PR for this after this one is merged.